radareorg/cutter.re
# Improving Decompiler Widget - GSoC project blog

**Detailed description**
Important things that are left are: 
1. An image for the post
2. My image thumbnail.


...

**Test plan**
1. Read and tell me if I  should improve it, and in what ways.
2. Make sure I haven't missed anything that's must be there in this.
3. Check if I made all the changes required for a new blog post.


...

**Closing issues**



...
